src/bufr/CMakeLists.txt

```diff
@@ -60,8 +67,8 @@ add_library( ${PROJECT_NAME}::ingester ALIAS ingester )
 add_executable(test_ingester.x test_ingester.cpp)
 target_link_libraries(test_ingester.x PRIVATE ${PROJECT_NAME}::ingester)
 
-add_executable(convertBufrToIoda.x main.cpp)
-target_link_libraries(convertBufrToIoda.x PRIVATE ${PROJECT_NAME}::ingester)
+add_executable(toIoda.x main.cpp)
+target_link_libraries(toIoda.x PRIVATE ${PROJECT_NAME}::ingester)
```

Comment:
toIoda.x is not explicit enough IMO. Can we at least make it Convert2Ioda.x or something like that?

Reply:
updated to convertToIoda.x